docs/design/design_v1.9.md

```diff
@@ -269,22 +317,26 @@ As soon as the control plane is available, kubeadm executes following actions:
 
 Please note that 
 
-1. Mark master phase can be invoked individually with the `kubeadm phase mark-master` command.
+1. Mark master phase can be invoked individually with the `kubeadm alpha phase mark-master` command.
 
 ### Configure TLS-Bootstrapping for node joining
 
 Kubeadm uses [Authenticating with Bootstrap Tokens](https://kubernetes.io/docs/admin/bootstrap-tokens/) for joining new nodes to an existing cluster; for more details see also [design proposal](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/cluster-lifecycle/bootstrap-discovery.md).
 
-`kubeadm init`  ensures that everything is properly configured for this process, and this includes following steps as well as setting API Server and controller flags as already described in previous paragraphs.
+`kubeadm init`  ensures that everything is properly configured for this process, and this includes following steps as well as setting API server and controller flags as already described in previous paragraphs.
+
+Please note that 
+
+1. TLS bootstrapping for nodes can be configured with the `kubeadm alpha phase bootstrap-token all` command, executing configuration steps described in following paragraphs; alternatively, each step can be invoked individually.
 
 #### Create a bootstrap token
 
 `kubeadm init`  create a first bootstrap token, either generated automatically or provided by the user with the `--token` flag; as documented in bootstrap token specification, token should be saved as secrets with name `bootstrap-token-<token-id>` under `kube-system` namespace.
 
 Please note that 
 
-1. The token will be used to validate temporary user during TLS bootstrap process; those users will be member of  `system:bootstrappers:kubeadm:default-node-token` group (nb. formerly `system:bootstrappers` in v1.7)
-2. Starting from 1.8 token has a limited validity, default 24Hours (that can be changed with `—token-ttl` flag)
+1. The token will be used to validate temporary user during TLS bootstrap process; those users will be member of  `system:bootstrappers:kubeadm:default-node-token` group.
+2. Token has a limited validity, default 24Hours (that can be changed with `—token-ttl` flag)
```

Comment:
_The_ token... (referencing "The default token created by kubeadm init" above)

default 24 hours (the interval may be changed with the `--token-ttl` flag)